src/lelantus.cpp

```diff
@@ -921,8 +944,8 @@ mint_info_container const & CLelantusState::Containers::GetMints() const {
     return mintedPubCoins;
 }
 
-std::unordered_map<uint256, GroupElement>&  CLelantusState::Containers::GetReducedHashToGroupElement() {
-    return reducedHashToGroupElement;
+std::unordered_map<uint256, lelantus::PublicCoin>&  CLelantusState::Containers::GetHashToGroupElement() {
```

Comment:
here we call the tag `hash`. And probably it should be called `tagToPublicCoin`?